src/Examples/Flowsheets/power_generation/ngcc/ngcc_soec.ipynb

```diff
@@ -519,109 +462,101 @@
-      "  15  5.2487028e+01 9.17e-01 4.62e+05  -1.0 2.04e+04    -  1.25e-01 7.14e-03f  4\n",
-      "  16  5.2567597e+01 9.37e-01 4.62e+05  -1.0 9.59e+04    -  1.50e-02 1.59e-03f  4\n",
-      "  17  5.1507424e+01 2.16e+00 4.37e+05  -1.0 8.02e+01  -4.9 6.06e-01 8.81e-02f  1\n",
-      "  18  5.1819258e+01 2.11e+00 2.40e+05  -1.0 4.72e+02    -  3.71e-01 4.47e-01f  1\n",
-      "  19  5.0961863e+01 2.73e+00 1.71e+02  -1.0 1.66e+03    -  9.90e-01 1.00e+00h  1\n",
+      "   0  5.1664952e+00 8.01e+00 1.28e+01  -1.0 0.00e+00    -  0.00e+00 0.00e+00   0\n",
+      "   1  5.1728493e+00 6.30e+00 1.15e+02  -1.0 1.32e+03    -  6.71e-01 2.14e-01h  1\n",
+      "   2  5.1760621e+00 5.61e+00 9.54e+01  -1.0 4.45e+02    -  5.78e-01 1.09e-01h  1\n",
+      "   3  5.1762395e+00 5.60e+00 3.89e+04  -1.0 5.34e+04    -  9.27e-01 1.86e-03h  1\n",
+      "   4  5.3886176e+00 7.06e-01 2.28e+04  -1.0 1.57e+05    -  3.08e-01 9.90e-01H  1\n",
+      "   5  5.3869023e+00 6.91e-01 2.23e+04  -1.0 5.70e+03    -  2.08e-02 2.09e-02f  2\n",
+      "   6  5.3882392e+00 6.63e-01 2.14e+05  -1.0 1.07e+03  -4.0 3.17e-01 4.06e-02h  1\n",
+      "   7  5.4827036e+00 5.89e-01 3.31e+05  -1.0 1.47e+03  -3.6 6.97e-01 8.48e-01H  1\n",
+      "   8  5.4413495e+00 5.52e-01 8.27e+05  -1.0 1.65e+04    -  3.24e-01 5.62e-02h  3\n",
+      "   9  5.4887674e+00 1.33e+00 3.76e+05  -1.0 2.11e+02  -4.1 6.80e-01 1.00e+00h  1\n",
       "iter    objective    inf_pr   inf_du lg(mu)  ||d||  lg(rg) alpha_du alpha_pr  ls\n",
-      "  20  5.1914326e+01 2.09e-01 3.41e-01  -1.0 1.40e+04    -  1.00e+00 1.00e+00h  1\n",
-      "  21  5.1380831e+01 1.41e+00 2.33e+05  -1.7 5.79e+04    -  5.33e-01 2.42e-01h  1\n",
-      "  22  4.8893192e+01 1.31e+01 1.93e+04  -1.7 1.38e+04    -  8.29e-01 8.64e-01f  1\n",
-      "  23  4.8930180e+01 1.17e+01 5.34e+04  -1.7 6.31e+03    -  6.72e-01 1.06e-01h  1\n",
-      "  24  4.9176459e+01 2.28e-01 9.60e+00  -1.7 3.55e+03    -  1.00e+00 1.00e+00h  1\n",
-      "  25  4.9225432e+01 1.96e-02 9.30e-03  -1.7 1.15e+02    -  1.00e+00 1.00e+00h  1\n",
-      "  26  4.9081760e+01 5.38e-01 2.99e+04  -3.8 7.73e+03    -  6.49e-01 4.98e-01h  1\n",
+      "  10  5.5317556e+00 1.21e+00 4.28e+05  -1.0 6.23e+03    -  5.28e-01 9.22e-02f  2\n",
+      "  11  5.5479811e+00 1.37e+00 4.82e+05  -1.0 5.05e+04    -  1.29e-01 6.51e-03f  3\n",
+      "  12  5.5742492e+00 2.01e-01 1.27e+05  -1.0 1.10e+02  -4.5 9.90e-01 8.53e-01F  1\n",
+      "  13  5.6249547e+00 7.68e-01 1.19e+05  -1.0 1.53e+04    -  2.75e-01 6.80e-02f  2\n",
+      "  14  5.6252496e+00 6.83e-01 1.08e+05  -1.0 2.21e+02  -5.0 9.90e-01 1.18e-01h  4\n",
+      "  15  5.6291898e+00 2.33e+00 9.32e+04  -1.0 1.27e+03  -5.5 1.07e-01 1.34e-01f  1\n",
+      "  16  5.5039846e+00 2.73e+00 2.61e+02  -1.0 2.35e+03    -  9.76e-01 1.00e+00h  1\n",
+      "  17  5.6853081e+00 6.16e+00 3.09e+01  -1.0 2.97e+03    -  8.87e-01 7.05e-01h  1\n",
       "Error in an AMPL evaluation. Run with \"halt_on_ampl_error yes\" to see details.\n",
```

Comment:
Are these warnings okay for the initial solve?

Reply:
If you know which side of the heat exchanger is going to be hot, it's easy to go into the heat exchangers and give `delta_temperature_in` and `delta_temperature_out` variables bounds of `(0, None)`, which should stop most AMPL errors. You still might get the occasional AMPL error because IPOPT can relax bounds, but it gets rid of most of the spam.